### ISSUE TITLE
Release New Version - Custom Properties

### ISSUE THREAD

**Issue by aarbro** [decalage2/olefile#132]
I see that support for custom properties was added quite a while back, but there has been no release. Is there any intent to do a new release anytime soon?

**Comment by kloczek** [comment]
+1

**Comment by decalage2** [comment]
Yes, it's on my todo list - so far I am spending more time on oletools releases...

**Comment by johnthagen** [comment]
It would also be great if the next release published wheel files (which are currently missing from the latest stable release).

Read more about the [advantages of wheels](https://pythonwheels.com/) at that link.
